src/main/java/io/vertx/mqtt/MqttEndpoint.java

```diff
@@ -252,6 +313,16 @@
   @Fluent
   MqttEndpoint publishCompletionHandler(Handler<Integer> handler);
 
+  /**
+   * Set the pubcomp handler on the MQTT endpoint. This handler is called when a PUBCOMP
+   * message is received by the remote MQTT client
+   *
+   * @param handler the handler
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  MqttEndpoint publishCompletionMessageHandler(Handler<MqttPubCompMessage> handler);
```

Comment:
I'm wondering if we should not bame it `pubCompMesssageHandler` for consistency

Reply:
What should we do with `MqttEndpoint publishCompletionHandler(Handler<Integer> handler);` then?

Reply:
right this is a good point.